frontend/dockerfile/docs/rules/invalid-default-arg-in-from.md

```diff
@@ -0,0 +1,40 @@
+---
+title: InvalidDefaultArgInFrom
+description: Using ARG with default value results in an empty or invalid base image name
+aliases:
+  - /go/dockerfile/rule/invalid-default-arg-in-from/
+---
+
+## Output
+
+```text
+Using ARG for FROM busybox:${TAG} with default value results in an empty or invalid base image name
```

Comment:
This should be something generic like "Global ARG with default values should produce a valid build"